Xamarin.AndroidBinderator/Engine.cs

```diff
@@ -150,6 +152,21 @@ static async Task DownloadArtifacts(MavenRepository maven, BindingConfig config,
 				using (var astrm = await mvnArt.OpenLibraryFile(mavenArtifact.Version, mavenProject.Packaging))
 				using (var sw = File.Create(artifactFile))
 					await astrm.CopyToAsync(sw);
+
+				// Determine MD5
+				try
+				{
+					// First try download
+					using (var astrm = await mvnArt.OpenLibraryFile(mavenArtifact.Version, mavenProject.Packaging + ".md5"))
```

Comment:
First try and download the MD5 from the source to have an additional assurance that the file we download is correct.

We could probably do better and confirm that the artifact is valid, and even hash the file we have on disk to avoid downloading the artifact again if this tool is run multiple times.